src/main/java/io/github/darealturtywurty/turtychemistry/core/init/ItemInit.java

```diff
@@ -55,10 +59,16 @@ public final class ItemInit extends AbstractInit {
             () -> new ChemistryItem(makeItemProperties()));
     public static final RegistryObject<ChemistryItem> RUBBER = ITEMS.register("rubber",
             () -> new ChemistryItem(makeItemProperties()));
+    public static final RegistryObject<ChemistryItem> BASIC_HAMMER = ITEMS.register("basic_hammer",
+            () -> new BasicHammer(new ChemistryItem.Builder(
+                    new Item.Properties().tab(TurtyChemistry.TAB).durability(70).stacksTo(1))));
+    public static final ImmutableMap<Supplier<Item>, Supplier<Item>> INGOT_SHEET_MAP = ImmutableMap.<Supplier<Item>, Supplier<Item>>builder()
+            .put(Ingots.ALUMINUM_INGOT::get, Sheets.ALUMINUM_SHEET::get).build();
```

Comment:
This is extremely janky. Do you think you could instead make this loadable through json? If not that's fine I will just do it after merge.

Reply:
well, I don't really know how Json parsing works in minecraft, so there might be a way to do it, but I don't know how to.

Reply:
ok thats fine. can you clear up the other comments and then i can merge

Reply:
I'll get on that